File: TFSWayUI/src/app/tasktracker/tasktracker.ts
export interface ITaskTracker {

    TaskTrackerID: number;
    ProjectID: number;
    Task: string;
    Responsible: string;
    PlanDate: Date;
    Status: string;
    CreatedDate: Date;
    CreatedBy: string;


    tasktrackers: Array<ITaskTracker>;

}

File: TFSWay.DomainModel/Entities/TaskTracker.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace TFSWay.DomainModel.Entities
{
   public class TaskTracker
    {
        [Key]
        public int TaskTrackerID { get; set; }
        public int ProjectID { get; set; }
        public string Task { get; set; }
        public string Responsible { get; set; }
        public string ResponsibleEmail { get; set; }
        public DateTime? PlanDate { get; set; }
        public string Status { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string CreatedBy { get; set; }
    }
}
